napari/_qt/qt_viewer.py

```diff
@@ -287,16 +287,23 @@ def _create_performance_dock_widget(self):
     def console(self):
         """QtConsole: iPython console terminal integrated into the napari GUI."""
         if self._console is None:
-            from .widgets.qt_console import QtConsole
-
-            self.console = QtConsole({'viewer': self.viewer})
+            try:
+                from napari_console import QtConsole
+
+                self.console = QtConsole(self.viewer)
+            except ImportError:
+                warnings.warn(
+                    'napari-console not found. It can be installed with'
+                    'pip install napari_console'
```

Comment:
```suggestion
                    ' "pip install napari_console"'
```
(note additional space there too)